samples/subsys/power/stm32_pm/sample.yaml

```diff
@@ -0,0 +1,17 @@
+sample:
+  name: Low Power Timer Sample for nucleo board
+tests:
+  sample.power.pm.stm32_lptim.lsi:
+    build_only: true
+    depends_on: lptim
+    filter: dt_compat_enabled_with_alias("gpio-leds", "led0") and
+            dt_compat_enabled_with_alias("gpio-leds", "led1") and
+            dt_compat_enabled("st,stm32-lptim")
+  sample.power.pm.stm32_lptim.lse:
+    build_only: true
+    depends_on: lptim
+    filter: dt_compat_enabled_with_alias("gpio-leds", "led0") and
+            dt_compat_enabled_with_alias("gpio-leds", "led1") and
+            dt_compat_enabled("st,stm32-lptim")
+    extra_configs:
+      - CONFIG_STM32_LPTIM_CLOCK_LSE=y
```

Comment:
Is it possible to exclude boards that don't have a LSE ?

Reply:
changed to have only 1 led : off = low power, on=running

Reply:
> Is it possible to exclude boards that don't have a LSE ?

@ABOSTM, it is not possible I think . But we're filtering on boards that declare `lptim` support and this should encompass LSE support.